lib/common/database.dart

```diff
@@ -563,8 +798,10 @@ class DatabaseService {
   /// Takes a query string and value to limit number of outputs
   /// Returns tasks that the query is a substring in, with amount specified by limit
   /// Return format is map of tasks
-  Future<Map<String, Task>> searchTaskDescription(String query, int limit) async {
-    final QuerySnapshot<Map<String, dynamic>> queriedTaskResults = await _substringQuery(query, limit, "tasks", "description");
+  Future<Map<String, Task>> searchTaskDescription(
+      String query, int limit) async {
+    final QuerySnapshot<Map<String, dynamic>> queriedTaskResults =
+        await _substringQuery(query, limit, "tasks", "description");
```

Comment:
This might conflict with mine. It looks like this was all formatting so i think use mine

Reply:
Yeah every time I save a file in VSC for flutter, it auto formats everything no matter what. I looked for how to stop it but came up dry :p